### PR TITLE
VZ-5361: Add acceptance test to check for valid service port names

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request adds an acceptance test to make sure service ports in the verrazzano-system namespace follow Istio naming conventions.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go [hunk 1]
```diff
@@ -528,3 +536,39 @@ func validateCorrectNumberOfPodsRunning(deployName string, nameSpace string) err
...
+			checkName := true
+			for _, exception := range exceptions {
+				if service.Name == exception {
+					checkName = false
+					break
+				}
+			}
+			if checkName {
+				hasPrefix := false
+				if strings.HasPrefix(port.Name, "http") || strings.HasPrefix(port.Name, "https") {
```
[the remark this answers is not in the document]
Comment (bobdonat) [reply tests/e2e/verify-install/verrazzano/verrazzano_test.go:566]: I originally had the hyphen but kiali names one of their ports http so I removed the hyphen

[the remark this answers is not in the document]
Comment (pfmackin) [reply tests/e2e/verify-install/verrazzano/verrazzano_test.go:566]: wonder if httpfoo would show a problem in Kiali.  Maybe you can check for exact match or prefix with hyphen

[the remark this answers is not in the document]
Comment (bobdonat) [reply tests/e2e/verify-install/verrazzano/verrazzano_test.go]: fixed

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go [hunk 2]
```diff
@@ -528,3 +536,40 @@ func validateCorrectNumberOfPodsRunning(deployName string, nameSpace string) err
...
+			checkName := true
+			for _, exception := range exceptions {
+				if service.Name == exception {
+					checkName = false
+					break
+				}
+			}
+			if checkName {
+				hasPrefix := false
+				if strings.HasPrefix(port.Name, "http") || strings.HasPrefix(port.Name, "https") ||
```
Comment (pfmackin) [inline tests/e2e/verify-install/verrazzano/verrazzano_test.go:566]: shouldn't this be ( strings.Compare("http") == 0 || strings.Compare("https") == 0  || ...

**Review by pfmackin — APPROVED** [review]

**Review by kminder — APPROVED** [review]
Approved.

File: tests/e2e/verify-install/verrazzano/verrazzano_test.go (hunk 2 above)
Comment (bobdonat) [reply tests/e2e/verify-install/verrazzano/verrazzano_test.go]: Fixed

**Pull request merged**